test-suite/coq-makefile/local-late-extension/run.sh

```diff
@@ -2,14 +2,13 @@
 
 . ../template/path-init.sh
 
+rm -rf _test
+mkdir _test
+find . -maxdepth 1 -not -name . -not -name _test -exec cp -r '{}' -t _test ';'
+cd _test
+
 set -x
 coq_makefile -Q . Top foo.v -o Makefile
-rm -f .*.d *.d
-make --no-print-directory -j1 printvo 2>&1 \
-    | grep -v 'make[^:]*: warning: -jN forced in submake: disabling jobserver mode.' \
-    | grep -v 'coq_makefile -Q . Top foo.v -o Makefile' \
-    | grep -v 'make[^:]*: Entering directory' \
-    | grep -v 'make[^:]*: Leaving directory' \
-    | grep -v 'exit 1' \
-           > make-printvo.log
-diff -u make-printvo.log.expected make-printvo.log || exit $?
+rm -f .*.d ./*.d
+make --no-print-directory -j1 printvo
```

Comment:
The `rm` is no longer necessary now that we're not taking the entire log; it was there only to normalize the log with regard to emitting info about generating .d files.
```suggestion
make --no-print-directory -j1 printvo
```

Reply:
```suggestion
make printvo
```
We actually also don't care about `-j1` and `--no-print-directory`.  They don't hurt anything, but they're no longer needed now that we're outputting to a file.